src/components/CreateABookForm.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from "prop-types";
+import { useState } from "react";
+
+const CreateABookForm = ({ handleAddBookItem }) => {
+  const [title, setTitle] = useState("");
+  const [author, setAuthor] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim() && author) {
+      handleAddBookItem(title, author);
+      setTitle("");
+      setAuthor("");
+    }
+  };
+
+  return (
+    <div className="form-wrapper">
+      <h1>ADD NEW BOOK</h1>
+      <form onSubmit={handleSubmit} className="book-form">
+        <input
+          type="text"
+          placeholder="Book title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <select value={author} onChange={(e) => setAuthor(e.target.value)}>
+          <option>Select Author</option>
+          <option value="Suzanne Collins">Suzanne Collins</option>
+          <option value="Frank Herbert">Frank Herbert</option>
+        </select>
```

Comment:
- [ ] Great job on creating the form so far but as mentioned in the requirements you should replace the category with author input so in the design you can see an input for a category so please replace it with an author. 🙏🏻🔥🤩

![image](https://github.com/DebyGrey/bookstore/assets/58553711/07178ff0-469b-4c8c-ae99-e68f7511216a)